Index: xkcd-generate-0.0.7/xkcd-generate.pl
===================================================================
--- xkcd-generate-0.0.7.orig/xkcd-generate.pl
+++ xkcd-generate-0.0.7/xkcd-generate.pl
@@ -20,7 +20,8 @@ my $VERSION = '0.0.7';
 my @final_books;
 
 # We'll keep this up here to ensure folks can change it if need be
-my $book_location = '/usr/share/doc/xkcd-generate';
+# my $book_location = '/usr/share/doc/xkcd-generate';
+my $book_location = '/usr/share/perl5/xkcd-generate';
 
 use Gtk3 '-init';
 use Glib 'TRUE', 'FALSE';
