Index: xkcd-generate-0.0.6/xkcd-generate.pl
===================================================================
--- xkcd-generate-0.0.6.orig/xkcd-generate.pl
+++ xkcd-generate-0.0.6/xkcd-generate.pl
@@ -19,7 +19,7 @@ use Getopt::Long;
 # have the first one (like CentOS), so use
 # the second one (MT::Auto).
 # use Math::Random::Secure 'irand';    # faster than rand
-use Math::Random::MT::Auto 'irand';
+# use Math::Random::MT::Auto 'irand';
 $| = 1;
 
 my $VERSION = '0.0.6';
@@ -217,7 +217,7 @@ sub generate {
         }
     }
     if ( $use_numbers ) {
-        my $irand = irand( 100 ) + 1;
+        my $irand = int(rand( 100 )) + 1;
         $irand = sanity( $irand );
         $entry_5->set_text( $irand );
         $final_sofar .= ' ' . $irand;
